oxygen-account/src/test/java/com/oxygenxml/account/user/UserControllerTest.java

```diff
@@ -303,5 +312,134 @@ void testChangeEmptyName() throws Exception {
 		.andExpect(jsonPath("$.errors[?(@.fieldName == 'name')].errorMessage", hasItem(Message.EMPTY_FIELD.getMessage())))
         .andExpect(jsonPath("$.errors[?(@.fieldName == 'name')].messageId", hasItem(Message.EMPTY_FIELD.getId())));
 	}
+	
+	/**
+	 * testChangePassword tests the situation when a user want to change his password
```

Comment:
Nu trebuie inclus numele metodei in JavaDoc.
Descrierea metodei trebuie sa inceapa cu majuscula.